File: backend/app/schemas/enbloc.py
from pydantic import BaseModel


class DevelopmentProfile(BaseModel):
    id: int
    name: str
    district: str
    age_years: int
    plot_ratio_headroom: float
    land_area_sqft: float
    previous_csc_attempt: bool
    ownership_units: int
    lease_remaining_years: int
    csc_status: str
    lat: float
    lng: float
    score: int

File: frontend/src/api/enbloc.ts
import { apiGet } from './client'

export interface DevelopmentProfile {
  id: number
  name: string
  district: string
  age_years: number
  plot_ratio_headroom: number
  land_area_sqft: number
  previous_csc_attempt: boolean
  ownership_units: number
  lease_remaining_years: number
  csc_status: string
  lat: number
  lng: number
  score: number
}

export function getDevelopments(): Promise<DevelopmentProfile[]> {
  return apiGet('/api/enbloc/developments')
}
